test/integration/support/testUtils.ts

```diff
@@ -9,38 +9,38 @@ export class MockPublisher {
     this.ably = new Ably.Realtime.Promise({ key, clientId: 'mock-publisher', environment: 'sandbox' });
   }
 
-  sendRawMessage = async (channel: string, message: unknown): Promise<void> => {
-    await this.ably.channels.get(channel).publish(EventNames.Raw, message);
+  sendRawMessage = async (trackingId: string, message: unknown): Promise<void> => {
+    await this.ably.channels.get(`tracking:${trackingId}`).publish(EventNames.Raw, message);
```

Comment:
I would be tempted to pull `'tracking'` up to a constant somewhere, but it's not a biggie (especially given these are "just" test utils).

Reply:
Good idea, I've refactored this a bit in 726539b